tests/foreman/ui/test_provisioningtemplate.py

```diff
@@ -96,8 +98,7 @@ def test_positive_end_to_end(session, module_org, module_location, template_data
     name = gen_string('alpha')
     new_name = gen_string('alpha')
     os = entities.OperatingSystem().create()
```

Comment:
Since this is the last occurrence of `entities`, would you mind changing it to `target_sat.api`?